### DISCUSSION TITLE
Suggestion: Make the Loot Tracker act like the Mobile Loot Tracker

### DISCUSSION THREAD

**Discussion by ghost** [runelite/runelite#17584] [Ideas]
Adds the ability to see pets on the loot tracker
Adds the ability to see items from skilling activities (MLM, Ores, Farming, Woodcutting, etc)
I feel like the mobile loot tracker is so much better than RuneLite loot tracker and I wish they could be the same
![image](https://github.com/runelite/runelite/assets/73009020/33c3cb36-3f9b-46f4-9279-86e8c1da74b4)

**Comment by Felanbird** [comment]
The general consensus since 2018 is mining/woodcutting is not loot, and it would be a huge hassle to implement such a feature from where RuneLite stands. 
Jagex is easily able to create a "better" loot tracker, due to the fact that they have all the access to all the data they would ever need, for such a feature.
